### PR TITLE
Finalizes linting configurations and adds style configuration to build step

### PR DESCRIPTION
**Pull request by superbuggy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
     "vibe-check": "npx vite-bundle-visualizer",
     "generate-openapi-client": "openapi-generator-cli generate -i osidb-3.4.0.oas3.json --output src/generated-client -g typescript-axios",
     "mock-osidb-server": "prism mock osidb-3.0.0.oas3.json",
-    "lint": "eslint . --ext .js,.ts,.vue"
+    "lint": "yarn lint-style && yarn lint-ts-vue",
+    "lint-ts-vue": "eslint . --ext .js,.ts,.vue",
+    "lint-style": "npx stylelint '**/*.{vue,css,scss}'"
```
Comment (JakubFrejlach) [inline package.json:25]: What is the difference between these two?

**Review by JakubFrejlach — APPROVED** [review]
One small question, otherwise LGTM! Hopefully this is the last big reformatting.

File: package.json (hunk 1 above)
Comment (C-Valen) [reply package.json:25]: First one (eslint) is for linting ts/js code and second one (stylelint) is for css/scss code

Comment (JakubFrejlach) [reply package.json:25]: Thanks!

**Review by d10n — APPROVED** [review]

**Pull request merged**